### DISCUSSION TITLE
Working of set_initial_soc= (0,1)

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3983] [Q&A]
I wanted to know how the model calculates the starting voltage or starting capacity when we set the initial_soc=, let say (0.3).
Is there a mathematical formulation related to set_initial_soc=some value?

TIA

**Comment by valentinsulzer** [comment]
It is explained in https://docs.pybamm.org/en/latest/source/examples/notebooks/models/electrode-state-of-health.html
